docs/src/usage/cookbook.md

```diff
@@ -118,18 +118,83 @@ rs = ADRIA.run_scenarios(dom, p_df, "45")
 ```julia
 using ADRIA
 using ADRIA: run_site_selection
-using DataFrames
 
+@info "Loading data package"
+here = @__DIR__
+dom = ADRIA.load_domain(joinpath(here, "Moore_2023-08-17"), "45")
+scens = ADRIA.sample_site_selection(dom, 8) # Get scenario dataframe.
 
```

Comment:
Replace `@info` with comments

Make the example more general, assume users do not have the specific data package you are using.

`scens = ADRIA.sample_site_selection(dom, 8) # Get scenario dataframe.`

As mentioned before, **two** spaces before an inline comment please (we're adopting PEP8-style recommendations here).

But is this comment really necessary?
Is the returned dataframe really scenarios?